concepts/arrays/about.md

```diff
@@ -71,4 +71,4 @@ describe [5; 7]     // => "Array with two items (first item ignored)"
 describe [5; 7; 9]  // => "Array with many items (all items ignored)"
```

Comment:
This section also contains some lists. Could you also update those?

Reply:
Yep, done!